src/content/getting-started/install-an-application/_index.md

```diff
@@ -15,4 +15,238 @@ user_questions:
+test01-aws-pod-identity-webhook        1.16.0              12m          117s            deployed
+...
+test01-observability-bundle            1.5.2               12m          4m28s           deployed
+test01-security-bundle                 1.8.0               12m          3m32s           deployed
+test01-teleport-kube-agent             0.9.2               12m          4m24s           deployed
+test01-vertical-pod-autoscaler         5.2.4               12m          110s            deployed
+```
+
+__Note__: We don't enforce the cluster prefix (here: `test01-`), but it's a good practice to have it.
+
+You can see that several applications already exist for the workload cluster `test01`. Most of the apps run directly in the workload cluster itself. The operator which actually deploys the applications to the workload clusters is running on the management cluster and is called `app-operator`. Learn more about this process [in this guide]({{< relref "/vintage/platform-overview/app-platform/" >}}).
+
+## Step 2: Install an ingress nginx controller
+
+Before we install our publicly accessible `hello-world` app, we need to have an ingress controller running in the cluster. The ingress controller is responsible for routing the incoming traffic to the correct service in the cluster and make it available to the public.
+
+To know which applications are available for customers we've extended the platform with two custom resources. First resource is the [`AppCatalog`]({{< relref "/vintage/use-the-api/management-api/crd/appcatalogs.application.giantswarm.io/" >}}) which is a recipient to collect application definitions that are available to install in the workload clusters. The second is the [`AppCatalogEntry`]({{< relref "/vintage/use-the-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}}) which is the representation of the application definition which has a version defined.
+
+By default there is a single catalog in the platform with the applications maintained by us:
+
+```nohighlight
+kubectl gs get catalogs
+NAME                    CATALOG URL
+giantswarm              https://giantswarm.github.io/giantswarm-catalog/
+```
+
+__Note__: You can [create your own catalog]({{< relref "/vintage/getting-started/app-platform/create-catalog/" >}}) and add your own applications to it.
+
+To browse which applications are available in the catalog you can run the following command:
+
+```nohighlight
+kubectl get appcatalogentries -n default -l application.giantswarm.io/catalog=giantswarm
+````
+
+In our case we're interested in the ingress nginx controller, so let's check the latest version available in the catalog:
+
+```nohighlight
+kubectl get appcatalogentries -n default -l app.kubernetes.io/name=ingress-nginx
+NAME                             CATALOG      APP NAME        VERSION   UPSTREAM VERSION   AGE
+giantswarm-ingress-nginx-3.9.2   giantswarm   ingress-nginx   3.9.2     1.11.2             2d4h
+...
+```
```

Comment:
Well, since this is an introduction, it is useful to introduce the concept of catalogue and entry. And I favour using a single command with a flag to filter then chaining tools. That's probably something we need to agree

Reply:
Sure, what I wrote are only suggestions. Please resolve any of my comments which you think are done/unneeded. Then I can go on reviewing and approving this.